core/src/apps/nem/mosaic/layout.py

```diff
@@ -76,21 +77,16 @@ def _supply_message(supply_change):
 
 async def _require_confirm_properties(ctx, definition: NEMMosaicDefinition):
     # TODO: we should send a button request here
-    properties = _get_mosaic_properties(definition)
-    first_page = const(0)
-    paginator = paginate(_show_page, len(properties), first_page, properties)
-    await ctx.wait(paginator)
-
-
-@ui.layout
-async def _show_page(page: int, page_count: int, content):
-    content = Scrollpage(content[page], page, page_count)
-    if page + 1 == page_count:
-        if await ConfirmDialog(content) != CONFIRMED:
-            raise wire.ActionCancelled("Action cancelled")
+    pages = _get_mosaic_properties(definition)
```

Comment:
this looks like it could use `require_confirm` directly

Reply:
See the comment above, it is a backward-incompatible change.